package.json

```diff
@@ -129,9 +129,12 @@
     ]
   },
   "devDependencies": {
+    "react-json-view": "chacha912/react-json-view#master",
```

Comment:
@chacha912

Yorkie UI is currently using React 18. However, the forked version of react-json-view only supports up to React 17. This version mismatch is causing an overriding peer dependency error. Given this situation, I'm wondering if we should update the forked react-json-view to resolve the issue.

```sh
$ npm install

npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: flux@4.0.4
npm WARN Found: react@18.3.1
npm WARN node_modules/react
npm WARN   react@"^18.3.1" from the root project
npm WARN   18 more (@ark-ui/react, @reduxjs/toolkit, ...)
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer react@"^15.0.2 || ^16.0.0 || ^17.0.0" from flux@4.0.4
npm WARN node_modules/flux
npm WARN   flux@"^4.0.1" from react-json-view@1.21.3
npm WARN   node_modules/react-json-view
npm WARN
npm WARN Conflicting peer dependency: react@17.0.2
npm WARN node_modules/react
npm WARN   peer react@"^15.0.2 || ^16.0.0 || ^17.0.0" from flux@4.0.4
npm WARN   node_modules/flux
npm WARN     flux@"^4.0.1" from react-json-view@1.21.3
npm WARN     node_modules/react-json-view
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: react-json-view@1.21.3
npm ERR! Found: react@18.3.1
npm ERR! node_modules/react
npm ERR!   react@"^18.3.1" from the root project
npm ERR!   peer react@">=18.0.0" from @ark-ui/react@2.2.3
npm ERR!   node_modules/@ark-ui/react
npm ERR!     @ark-ui/react@"^2.2.3" from yorkie-ui@0.5.16
npm ERR!     node_modules/yorkie-ui
npm ERR!       yorkie-ui@"^0.5.16" from the root project
npm ERR!   17 more (@reduxjs/toolkit, @testing-library/react, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^17.0.0 || ^16.3.0 || ^15.5.4" from react-json-view@1.21.3
npm ERR! node_modules/react-json-view
npm ERR!   dev react-json-view@"chacha912/react-json-view#master" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: react@17.0.2
npm ERR! node_modules/react
npm ERR!   peer react@"^17.0.0 || ^16.3.0 || ^15.5.4" from react-json-view@1.21.3
npm ERR!   node_modules/react-json-view
npm ERR!     dev react-json-view@"chacha912/react-json-view#master" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR!
npm ERR! For a full report see:
npm ERR! /Users/hackerwins/.npm/_logs/2024-06-24T02_12_21_273Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: /Users/hackerwins/.npm/_logs/2024-06-24T02_12_21_273Z-debug-0.log
```

Reply:
I updated the forked `react-json-view` repository to the latest version. 
It seems that we can replace it with a tree view component in the future.
- https://park-ui.com/docs/panda/components/tree-view
- https://mui.com/x/react-tree-view/simple-tree-view/customization/#connection-border